org.freedesktop.Sdk.Extension.php73.json

```diff
@@ -98,28 +96,60 @@
 				"--enable-xmlrpc",
 				"--enable-xmlwriter",
 				"--enable-xsl",
-				"--with-libzip",
 				"--enable-zlib",
-				"--with-zlib",
-				"--with-openssl"
+				"--with-argon2",
+				"--with-config-file-scan-dir=${FLATPAK_DEST}/lib/etc:/app/etc/php:/var/config/php/7.3/ini",
```

Comment:
Why `${FLATPAK_DEST}/lib/etc` and not just `${FLATPAK_DEST}/etc`?

Reply:
Because the default php.ini is in lib.

I thought about putting it in etc/php to match with /app/etc/php, but then the path would be /usr/lib/sdk/php73/etc/php

Just trying to find some uniformity.

Reply:
What's wrong with `/usr/lib/sdk/php73/etc/php`? It's immutable anyway.

Reply:
Thinking about it again, it is better.